Developer-Guide.md

```diff
@@ -301,6 +310,173 @@ $ sudo systemctl restart docker
+# container workloads. This is an optional setting, except if
+# default_container_trust is set to "untrusted".
+runtime_untrusted_workload = "/usr/local/bin/kata-runtime"
+
+# default_workload_trust is the default level of trust crio puts in container
+# workloads. It can either be "trusted" or "untrusted", and the default
+# is "trusted".
+# Containers can be run through different container runtimes, depending on
+# the trust hints we receive from kubelet:
+# - If kubelet tags a container workload as untrusted, crio will try first to
+# run it through the untrusted container workload runtime. If it is not set,
+# crio will use the trusted runtime.
+# - If kubelet does not provide any information about the container workload trust
+# level, the selected runtime will depend on the default_container_trust setting.
+# If it is set to "untrusted", then all containers except for the host privileged
+# ones, will be run by the runtime_untrusted_workload runtime. Host privileged
+# containers are by definition trusted and will always use the trusted container
+# runtime. If default_container_trust is set to "trusted", crio will use the trusted
+# container runtime for all containers.
+default_workload_trust = "untrusted"
+
+```
+
+Restart CRI-O to take changes into account
+```
+$ sudo systemctl restart crio
+```
+
+### CRI-containerd
+
+If you picked CRI-containerd, please follow the instructions
+[here](https://github.com/containerd/cri#getting-started-for-developers)
+to get it properly installed.
+
+And for the customization part so that CRI-containerd will pick Kata Containers
+runtime, please follow our internal documentation
+[here](https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#configure-containerd-to-use-kata-containers).
+
+## Install Kubernetes
+
+Depending on what your needs are and what you expect to do with Kubernetes,
+please refer to the following
+[documentation](https://kubernetes.io/docs/setup/pick-right-solution/) to
+install it correctly.
+
+Kubernetes talks with CRI implementations through a `container-runtime-endpoint`,
+also called CRI socket. This socket path is different depending on which CRI
+implementation you choosed, and the kubelet service has to be updated accordingly.
+
+### Configure for CRI-O
+
+_/etc/systemd/system/kubelet.service.d/0-crio.conf_
+```
+[Service]                                                 
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///var/run/crio/crio.sock"
+```
+
+### Configure for CRI-containerd
+
+_/etc/systemd/system/kubelet.service.d/0-cri-containerd.conf_
+```
+[Service]                                                 
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
+```
+For more details about CRI-containerd, click
```

Comment:
s/click [here]/see the [documentation(xxx)' or similar. We don't say 'click here' anywhere else I think.

Reply:
I agree with @grahamwhaley  - we shouldn't use "click here'. Maybe the following: 

For more information about CRI-containerd see the "Configure Kubelet to use containerd" documentation [here](https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#configure-kubelet-to-use-containerd).

Reply:
done